### PR TITLE
Improve README and CONTRIBUTING somewhat

### PR DESCRIPTION
**Pull request by michaelpj**
Simon was looking at these which prompted me to read them again and make a few changes.

### CODE REVIEW

**Comment by reactormonk** [comment]
TBH, I'd take a hard look on what exactly you could delete here as well - the shorter, the more likely people will actually read it.

**Comment by michaelpj** [comment]
Well, at the moment I'm trying to at least make sure everything is written down somewhere. Even if people don't read it all initially it can function as a reference. We've got too much knowledge that's just in people's heads at the moment.

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -122,6 +191,7 @@ and people may not be notified if you just push commits.
```
Comment (reactormonk) [inline CONTRIBUTING.adoc]: I'd add a short list of commands to execute before pushing (e.g. stylish, nix regenerate etc.) here.

Comment (michaelpj) [reply CONTRIBUTING.adoc]: Added, thanks!

**Pull request merged**